LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2021 Kings Distributed Systems
+Copyright (c) 2023 Distributive
```

Comment:
**todo:** This should match the company named in https://github.com/Distributive-Network/PythonMonkey/blob/main/LICENSE#L3.

```suggestion
Copyright (c) 2023 Distributive Corp.
```

Reply:
My bad! Thanks for pointing this out

Reply:
Added Corp. and added it to the commit

https://github.com/Distributive-Network/eslint-config/pull/47/files#diff-c693279643b8cd5d248172d9c22cb7cf4ed163a3c98c8a3f69c2717edd3eacb7